0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -1,73 +1,56 @@
 ---
 title: The Simple Payment Setup Protocol (SPSP)
-draft: 6
+draft: 7
 ---
 # Simple Payment Setup Protocol (SPSP)
 
 ## Preface
 
-This document describes the Simple Payment Setup Protocol (SPSP), a basic protocol for exchanging payment information between senders and receivers to facilitate payment over Interledger. SPSP uses the [STREAM](../0029-stream/0029-stream.md) transport protocol for condition generation and data encoding.
+This document describes the Simple Payment Setup Protocol (SPSP), a basic protocol for exchanging payment information between counterparties to facilitate payment over Interledger. SPSP uses the [STREAM](../0029-stream/0029-stream.md) transport protocol for condition generation and data encoding.
```

Comment:
If I'm not mistaken, @adrianhopebailie suggested "counterparties" because it is not sender and receiver in the sense of push payments and counterparties distinguishes it from push payments. 
@adrianhopebailie, any comments?